app/controllers/planets_controller.rb

```diff
@@ -7,6 +7,9 @@ def new
     @planet = Planet.new
   end
 
+  def edit
+  end
```

Comment:
I'm missing the @planet here (via a find method) 
(Probably can extracted in a before_action method)

Reply:
It was done in the previous pull request for the show